db/institutions.yml

```diff
@@ -107,7 +107,7 @@ esrf:
   aka: [European Synchrotron Radiation Facility, ESRF]
   city: Grenoble
   country: France
-  departments: {dubble: Dutch-Belgian Beamline}
+  departments: {db_bl: Dutch-Belgian Beamline, aka: [sas_bl]}
   name: European Synchrotron Radiation Facility
   state:
```

Comment:
we can't have empty keys.  Please put an empty string for the key, or remove the key.